test/cram/interactive_viewer_dummy.ml

```diff
@@ -0,0 +1,56 @@
+open Cmdliner
+
+let input_events : Notty.Unescape.event list ref = ref []
+let output_image : Notty.image ref = ref Notty.I.empty
+
+let open_dummy_terminal () : unit =
+  let input_fd = Unix.openfile "/dev/null" Unix.[O_RDWR] 0o666 in
+  let output_fd = Unix.dup Unix.stdout in
+  Unix.dup2 input_fd Unix.stdin;
+  Unix.dup2 output_fd Unix.stdout;
+  Unix.close input_fd;
+  Unix.close output_fd
+
+
+let output_image : Notty.image = !output_image
```

Comment:
This remember the very first image, which is empty. I think you mean to use `!output_image` in `print_image` instead of `output_image` and to remove this binding.